src/components/Dashboard.tsx

```diff
@@ -1,6 +1,52 @@
 import React from 'react';
 import {createStyles, makeStyles} from "@material-ui/core/styles";
 
+
+export interface LogsProps {
+    logsURL: string;
+}
+
+const useLogsDashboardStyles = makeStyles(() =>
+    createStyles({
+        root: {
+            width: "100%",
+            // TODO: We need to calculate the height
+            height: '800px'
+        },
+        logsDashboardIFrame: {
+            width: "100%",
+            maxHeight: '100%',
+            height: "100%",
+            border: 'none'
+        },
+    }),
+);
+
+export const LogsDashboard: React.FC<LogsProps> = ({logsURL}) => {
+    const classes = useLogsDashboardStyles();
+
+    return (
+        <div
+            className={classes.root}
+        >
+            <iframe
+                className={classes.logsDashboardIFrame}
+                src={logsURL}>
+            </iframe>
+        </div>
+    )
+
+};
+
+export function createLogsURL(baseLogsURL: string, parameters: Record<string, string | any>): string {
+    const filterKey = Object.keys(parameters)[0];
+    const filterVal = Object.values(parameters)[0];
+    const a = "_a=(description:'',filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:logstash,key:KEY,negate:!f,params:(query:VALUE),type:phrase),query:(match_phrase:(KEY:VALUE)))),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),panels:!((embeddableConfig:(),gridData:(h:15,i:a637bcb2-4c05-4ad8-ad59-89cb37c9c66c,w:48,x:0,y:0),id:e0d473a0-a6c1-11ea-bc41-3318d1349021,panelIndex:a637bcb2-4c05-4ad8-ad59-89cb37c9c66c,type:search,version:'7.6.2')),query:(language:kuery,query:''),timeRestore:!f,title:'',viewMode:view)&query='KEY:VALUE'".replace(/VALUE/g, filterVal).replace(/KEY/g, filterKey);
```

Comment:
What do you think about move kibana board configuration into /configuration endpoint of ODAHU API, so avoid hardcode it in UI code?

Reply:
Yes, it have to be done but it's not only about URI but also about panelIndex and id taht you can see in `a` string so right now I have no idea how to do it fine.